src/ResourceManager/RecoveryServices.Backup/RecoveryServices.Backup/Models/IaasVmModels/Container.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ProviderModel.Models.IaasVmModels
```

Comment:
no need to drill down upto IaasVmModels - Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ProviderModel.Models